yodeploy/util.py

```diff
@@ -41,6 +41,8 @@ def extract_tar(tarball, root):
     directory to root.
     """
     workdir = os.path.dirname(root)
+    if sys.version_info[0] < 3:
+        workdir = workdir.encode(sys.getfilesystemencoding())
```

Comment:
That's already happening, I believe.

https://docs.python.org/2.7/library/tarfile.html#tarfile.ENCODING

Reply:
That's the encoding of paths, within the tarfile. Not exactly the same thing. Nothing is encoding or decoding this parameter, before attempting to join it with names from the tarfile.